.github/workflows/deploy.yaml

```diff
@@ -1,106 +1,79 @@
-        uses: docker/setup-buildx-action@v1
-
-      - name: Login to DockerHub
-        uses: docker/login-action@v1
-        with:
-          username: ${{ secrets.DOCKER_USER_NAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
-
-      - name: Set Docker Metadata
-        id: meta
-        uses: docker/metadata-action@v3
-        with:
-          images: archwaynetwork/archwayd
-          tags: |
-            type=semver,pattern={{raw}}
-            type=sha
-
-
-      - name: Build and push docker image
-        uses: docker/build-push-action@v2
-        with:
-          push: true
-          tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
-          cache-from: type=gha
-          cache-to: type=gha,mode=max
-
-  deploy-titus:
-    runs-on: ubuntu-latest
-    if: ${{ github.event_name == 'workflow_dispatch' || github.event_name == 'push' && endsWith(github.ref, github.event.repository.default_branch) }}
-
-    needs:
-      - build-docker-image
-
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-
-      - name: Inject version and secrets to titus-1.yaml
-        run: |
-          sed -i -e "s|NODE_VERSION|${{ needs.build-docker-image.outputs.image-version }}|g" ci/titus-1.yaml
-          sed -i -e "s|TITUS_1_ACCOUNT_MNEMONIC_B64|${{ secrets.TITUS_1_ACCOUNT_MNEMONIC_B64 }}|g" ci/titus-1.yaml
-          sed -i -e "s|TITUS_1_FAUCET_ACCOUNT_MNEMONIC_B64|${{ secrets.TITUS_1_FAUCET_ACCOUNT_MNEMONIC_B64 }}|g" ci/titus-1.yaml
-          sed -i -e "s|TITUS_1_PRIV_VALIDATOR_KEY_B64|${{ secrets.TITUS_1_PRIV_VALIDATOR_KEY_B64 }}|g" ci/titus-1.yaml
-
-      - name: Deploy Chain
-        uses: steebchen/kubectl@v2.0.0
-        with:
-          config: ${{ secrets.GKE_SHARED_NONPROD_GH_ACTIONS_BOT_KUBECONFIG }}
-          command: apply -n testnets -f ci/titus-1.yaml
-
-  deploy-constantine:
-    runs-on: ubuntu-latest
-    if: ${{ github.event_name == 'workflow_dispatch' || github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v') }}
-
-    needs:
-      - build-docker-image
-
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-
-      - name: Get latest release tag
-        id: latesttag
-        uses: "WyriHaximus/github-action-get-previous-tag@v1"
-
-      - name: Update NODE_VERSION
-        run: |
-          sed -i -e "s|NODE_VERSION|${{ steps.latesttag.outputs.tag }}|g" ci/constantine-1.yaml
-
-      - name: Deploy Chain
-        uses: steebchen/kubectl@v2.0.0
-        with:
-          config: ${{ secrets.GKE_SHARED_NONPROD_GH_ACTIONS_BOT_KUBECONFIG }}
-          command: apply -n testnets -f ci/constantine-1.yaml
+# name: Build and Deploy
+#
+# on:
+#   push:
+#     branches: [main]
+#     tags: [v*]
+#
+#   pull_request:
+#     branches: [main]
+#
+#   workflow_dispatch:
+#     branches: [main]
+#
+# jobs:
+#   build-docker-image:
```

Comment:
Lets update the build part here and re-enable titus pipeline

Reply:
this is now implemented